From 87086cee978ec5dfe8fa0d5349fe5f0aa9419dea Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Mon, 8 Jul 2019 10:56:31 +0000
Subject: [PATCH] joyent/node-zkstream#9 use var instead of const so we can run
 in strict mode joyent/node-zkstream#29 unreliable tests assuming after sync()
 all watchers have fired joyent/node-zkstream#22 increase test coverage
 joyent/node-zkstream#24 remove watchers from index once all event handlers
 are gone

---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index ca63117..59e9005 100644
--- a/README.md
+++ b/README.md
@@ -1,4 +1,5 @@
 zkstream
+
 ========
 
 A minimal streams-based node client for the ZooKeeper protocol.
-- 
2.21.0

